code/modules/projectiles/guns/projectile/shotgun/ks23.dm

```diff
@@ -0,0 +1,21 @@
+/obj/item/gun/projectile/shotgun/pump/ks
+	name = "KS-23"
+	desc = "Karabin Spetsnaliy, desgined by Excelsior for a cheap crowd control for their agents, made with refused anti-aircraft barrels"
+	icon = 'icons/obj/guns/projectile/ks23.dmi'
+	icon_state = "ks23"
+	item_state = "ks23"
+	max_shells = 4
+	w_class = ITEM_SIZE_HUGE
+	force = WEAPON_FORCE_ROBUST
+	slot_flags = SLOT_BACK
+	matter = list(MATERIAL_PLASTEEL = 20, MATERIAL_PLASTIC = 10, MATERIAL_STEEL = 15)
+	proj_step_multiplier = 1.2
+	ammo_type = /obj/item/ammo_casing/shotgun/pellet/scrap
+	price_tag = 3000
```

Comment:
Are you sure this is a right price for that matter cost?

Reply:
how about 2000? the gladstone is 1800 for less material (i didnt put much though on the gun prices but thinking about it if the guild get their hands on it they could do a lot of profit indeed)

Reply:
Price also decreases spawn price and the such so there's a reason to increase price if its a good weapon.

Reply:
the moment i lower the price to 2K i get told to get it up xd, i leave it at 2K for now and i will wait for more intel to adjust the price, thanks a lot for the help.